.circleci/config.yml

```diff
@@ -700,6 +711,10 @@ workflows:
             -   snyk-test:
                     requires:
                         - dashboards-build
+                    filters:
+                        branches:
+                            ignore:
+                                - pull/*
```

Comment:
It seems like we should run snyk tests on forked PR branches as well.

Reply:
We don't pass tokens so that might be hard to do

Reply:
okay, at least any new vulnerabilities would theoretically be caught in main.

Reply:
If we changed the snyk scanner to just block PRs (and not use tokens), this might work.